lib/types.ts

```diff
@@ -0,0 +1,19 @@
+import { StoryblokConfig, StoryblokComponent } from "storyblok-js-client";
+
+declare global {
+  interface Window {
+    storyblokRegisterEvent: any;
+    StoryblokBridge: any;
+  }
+}
+
+export interface BlokData extends StoryblokComponent<string> {
+  [index: string]: any;
+}
+
+export interface SDKOptions {
```

Comment:
I would suggest to us to follow a pattern for the namespace here. We use `Sb<name>` for naming our own properties/components, it would be like `SbSDKOptions`. Sounds good?

Reply:
Sounds like a great idea! One thing about that: on storyblok-js-client I see the prefix used is `Storyblok<name>` like `StoryblokConfig` or `StoryblokComponent`. Shall we use that one to align?

Reply:
You can use `Storyblok<name>` or `Sb<name>`. We use them both, `Sb<name>` is mostly used under our Storyfront's codebase.